digdag-standards/src/main/java/io/digdag/standards/operator/HttpOperatorFactory.java

```diff
@@ -177,13 +180,36 @@ else if (user.isPresent()) {
                 request.header(header.getKey(), header.getValue());
             }
 
+            configureQueryParameters(request);
+
             ContentResponse response = PollingRetryExecutor.pollingRetryExecutor(state, "request")
                     .withErrorMessage("HTTP request failed")
                     .run(s -> execute(request));
 
             return result(response, storeContent);
         }
 
+        private void configureQueryParameters(Request request)
+        {
+            // TODO: allow secret query parameters
+
+            try {
+                Config queryParameters = params.parseNestedOrGetEmpty("query");
```

Comment:
[minor] Just `query` can sound SQL's one to me. How about `query_string`? It's more explicit, a bit long though.